### DISCUSSION TITLE
Support for Vue 3?

### DISCUSSION THREAD

**Discussion by Mirta91** [fancyapps/ui#61] [General]
Are you planning the support for Vue 3 anytime soon?

**Comment by fancyapps** [comment]
Hi,

This is planned, but it is currently not possible to give any ETA.

**Comment by GiancarloGomez** [comment]
I have it working with Vue3 without doing anything special. I imported FancyBox on my main js file where I create my vue app and it handles the auto binding of galleries in views. For galleries that I need to start programmatically I just import Fancybox in the file. This might not be the right way to do it but it is working for now.

**Reply by Mirta91** [reply]
Thx, yup, I've managed to do the same. https://stackoverflow.com/a/68865178/4038579
